content/articles/okta-identity-provider.markdown

```diff
@@ -0,0 +1,97 @@
+---
+title: Okta as an Identity Provider
+excerpt: How to use Okta to create and log in to accounts.
+categories:
+- Account
+---
+
+# Okta as an Identity Provider
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Using Okta as an identity provider streamlines the login experience for you and your team.
+
+## Prequisites
+
+In order to proceed with configuring login with SSO through Okta, you must:
+
+- Have administrator access to an Okta organization
+- Have administrator access to a DNSimple account
+- Have the Okta integration feature enabled on your DNSimple account
+
+## Supported Features
+
+- **SP-initiated Single Sign-On (SSO)**: This authentication flow occurs when the user attempts to log in to DNSimple from Okta.
+- **Import Users**: Manage who can access your DNSimple account by assigning users to the Okta application
+- **Single Log Out (SLO)**: Users can log out of their DNSimple session by logging out of their Okta session.
+
+## Logging in via Okta
+
+You'll need to [link an Okta organization to your DNSimple account](#linking-an-okta-organization-to-a-dnsimple-account) before your team members can log in via Okta SSO.
+
+Once a Okta organization is linked to a DNSimple account, any member of the Okta organization can request access to the DNSimple account. They can do so by attempting to log in via Okta using the respective Okta domain.
+
+Once they are granted access, they will be able to log in directly via Okta. If the user logs out of Okta, they will be logged out of DNSimple.
+
+## Linking an Okta organization to a DNSimple account
+
+Follow the instructions below to connect DNSimple to your Okta organization.
+
+### Add the DNSimple App to Okta
+
+1. Login to your organization's Okta marketplace.
+1. Search for DNSimple, and then click Add.
+1. Enter an Application Label in General Settings.
+1. Click Done.
+1. Assign yourself to the application.
+1. Take note of your Client ID, Client Secret, and Okta Domain as you'll need it in the next step.
+
+### Configure DNSimple for Okta SSO
+
+1. At DNSimple, go to the <label>Account</label> page, and click the <label>Single Sign-On</label> tab.
+1. Click <label>Authorize with Okta</label> to authorize the Workspace. You'll need to log in to the Okta Workspace with access to the `okta.eventHooks.manage`  and `okta.eventHooks.read` scopes.
+1. After logging in via Okta, you will arrive back on the DNSimple Single Sign-On page with Okta SSO enabled for the account.
+1. Repeat the process for each account where you want Okta SSO enabled.
+
+If the Okta app is no longer linked (e.g. the access token is revoked), you can re-link the Okta application using the "Reauthorize with Okta" button.
```

Comment:
What do you think of this example `(e.g. the event hook is removed)` ?

Reply:
Should we mention that an event hook is created for the Okta org in https://dev-02367705-admin.okta.com/admin/workflow/eventhooks for the first linked account, and to refrain from altering the hook's configuration (although "Reauthorize with Okta" takes care of this")?